visual-regressions/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM abykovssc/visual-regressions
```

Comment:
Shouldn't we add it to SSC docker registry?

Reply:
we have an image in a private registry, just not in public one, and I'm not sure we even have a public registry 🤔